### PR TITLE
surface code of calculator

### PR DESCRIPTION
**Pull request by Lisa-467**
surface code of calculator

### CODE REVIEW

**Review by Pooja98Patil — COMMENTED** [review]
Add heading

**Review by Lisa-467 — COMMENTED** [review]
added heading as per request

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=h2, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h2>Calculator</h2>
+    <h2>Study of Github</h2>
```
Comment (Pooja98Patil) [inline index.html:10]: change heading to h1

Comment (Lisa-467) [reply index.html:10]: done changes

**Pull request merged**